index.js

```diff
@@ -48,7 +67,25 @@
 // 7 - 12 months 4% of their body weight
 
 // when you are finished invoke your function with the weight of 15 lbs and the age of 1 year - if your calculations are correct your result should be 0.44999999999999996
-  
+function feedMeSeymour(dogMonths, dogWeight) {
```

Comment:
For if/else statements. the structure should be:

```
if (some statement) {
return this;
} else if (some statement) {
return this;
} else if (some statement) {
return this;
} else {
return this;
}
```
notice how its not : 
if 
if 
if